fluent-bit-cloudwatch.go

```diff
@@ -59,79 +58,82 @@ func getPluginInstance(ctx unsafe.Pointer) *cloudwatch.OutputPlugin {
 }
 
 func getConfiguration(ctx unsafe.Pointer, pluginID int) cloudwatch.OutputPluginConfig {
 	config := cloudwatch.OutputPluginConfig{}
 	config.PluginInstanceID = pluginID
+
 	config.LogGroupName = output.FLBPluginConfigKey(ctx, "log_group_name")
 	logrus.Infof("[cloudwatch %d] plugin parameter log_group = '%s'", pluginID, config.LogGroupName)
+
 	config.LogStreamPrefix = output.FLBPluginConfigKey(ctx, "log_stream_prefix")
 	logrus.Infof("[cloudwatch %d] plugin parameter log_stream_prefix = '%s'", pluginID, config.LogStreamPrefix)
+
 	config.LogStreamName = output.FLBPluginConfigKey(ctx, "log_stream_name")
 	logrus.Infof("[cloudwatch %d] plugin parameter log_stream_name = '%s'", pluginID, config.LogStreamName)
+
 	config.Region = output.FLBPluginConfigKey(ctx, "region")
 	logrus.Infof("[cloudwatch %d] plugin parameter region = '%s'", pluginID, config.Region)
+
 	config.LogKey = output.FLBPluginConfigKey(ctx, "log_key")
 	logrus.Infof("[cloudwatch %d] plugin parameter log_key = '%s'", pluginID, config.LogKey)
+
 	config.RoleARN = output.FLBPluginConfigKey(ctx, "role_arn")
 	logrus.Infof("[cloudwatch %d] plugin parameter role_arn = '%s'", pluginID, config.RoleARN)
+
 	config.NewLogGroupTags = output.FLBPluginConfigKey(ctx, "new_log_group_tags")
 	logrus.Infof("[cloudwatch %d] plugin parameter new_log_group_tags = '%s'", pluginID, config.NewLogGroupTags)
+
 	config.LogRetentionDays, _ = strconv.ParseInt(output.FLBPluginConfigKey(ctx, "log_retention_days"), 10, 64)
 	logrus.Infof("[cloudwatch %d] plugin parameter log_retention_days = '%d'", pluginID, config.LogRetentionDays)
+
 	config.CWEndpoint = output.FLBPluginConfigKey(ctx, "endpoint")
 	logrus.Infof("[cloudwatch %d] plugin parameter endpoint = '%s'", pluginID, config.CWEndpoint)
+
 	config.STSEndpoint = output.FLBPluginConfigKey(ctx, "sts_endpoint")
 	logrus.Infof("[cloudwatch %d] plugin parameter sts_endpoint = '%s'", pluginID, config.STSEndpoint)
+
 	config.CredsEndpoint = output.FLBPluginConfigKey(ctx, "credentials_endpoint")
 	logrus.Infof("[cloudwatch %d] plugin parameter credentials_endpoint = %s", pluginID, config.CredsEndpoint)
+
 	config.LogFormat = output.FLBPluginConfigKey(ctx, "log_format")
 	logrus.Infof("[cloudwatch %d] plugin parameter log_format = '%s'", pluginID, config.LogFormat)
 
 	return config
 }
 
-func getBoolParam(ctx unsafe.Pointer, param string, defaultVal bool) bool {
-	val := strings.ToLower(output.FLBPluginConfigKey(ctx, param))
-	if val == "true" {
-		return true
-	} else if val == "false" {
-		return false
-	} else {
-		return defaultVal
-	}
-}
-
-//export FLBPluginInit
+// FLBPluginInit is exported for fluent-bit.
 func FLBPluginInit(ctx unsafe.Pointer) int {
-	plugins.SetupLogger()
+	setupLogger()
 
 	err := addPluginInstance(ctx)
 	if err != nil {
-		logrus.Error(err)
+		logrus.Errorln("addPluginInstance():", err)
 		return output.FLB_ERROR
 	}
+
 	return output.FLB_OK
 }
 
-//export FLBPluginFlushCtx
```

Comment:
Same here. This `export` comment has syntactical meaning. We cannot change them. This is a `cgo` thing which tells this function will be used from C code I believe.